content/docs/traffic-management/route-delegation/label.md

```diff
@@ -0,0 +1,461 @@
+   ```sh
+   curl -i http://$INGRESS_GW_ADDRESS:8080/anything/team1/foo \
+   -H "host: delegation.example:8080" 
+   ```
+   {{% /tab %}}
+   {{% tab tabName="Port-forward for local testing" %}}
+   ```sh
+   curl -i localhost:8080/anything/team1/foo \
+   -H "host: delegation.example" 
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}
+
+   Example output: 
+   ```
+   HTTP/1.1 200 OK
+   access-control-allow-credentials: true
+   access-control-allow-origin: *
+   content-type: application/json; encoding=utf-8
+   content-length: 509
+   x-envoy-upstream-service-time: 0
+   server: envoy
+
+   {
+     "args": {},
+     "headers": {
+       "Accept": [
+         "*/*"
+       ],
+       "Host": [
+         "delegation.example:8080"
+       ],
+       "User-Agent": [
+         "curl/8.7.1"
+       ],
+       "X-Envoy-Expected-Rq-Timeout-Ms": [
+         "15000"
+       ],
+       "X-Forwarded-Proto": [
+         "http"
+       ],
+       "X-Request-Id": [
+        "65927858-2c6b-42ae-9278-8ff9d8bba3f8"
+       ]
+     },
+     "origin": "10.0.64.27:49526",
+     "url": "http://delegation.example:8080/anything/team1/foo",
+     "data": "",
+     "files": null,
+     "form": null,
+     "json": null
+   }
+   ```
+   
+6. Send a request to the `delegation.example` domain along the `/anything/team2/bar` path. Verify that you also get back a 200 HTTP response code.  
+   {{< tabs items="LoadBalancer IP address or hostname,Port-forward for local testing" >}}
+   {{% tab  %}}
+   ```sh
+   curl -i http://$INGRESS_GW_ADDRESS:8080/anything/team2/bar \
+   -H "host: delegation.example:8080" 
+   ```
+   {{% /tab %}}
+   {{% tab  %}}
+   ```sh
+   curl -i localhost:8080/anything/team2/bar \
+   -H "host: delegation.example:8080" 
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}
+
+   Example output: 
+   ```
+   HTTP/1.1 200 OK
+   access-control-allow-credentials: true
+   access-control-allow-origin: *
+   content-type: application/json; encoding=utf-8
+   content-length: 509
+   x-envoy-upstream-service-time: 1
+   server: envoy
+
+   {
+     "args": {},
+     "headers": {
+       "Accept": [
+        "*/*"
+       ],
+       "Host": [
+         "delegation.example:8080"
+       ],
+       "User-Agent": [
+         "curl/8.7.1"
+       ],
+       "X-Envoy-Expected-Rq-Timeout-Ms": [
+         "15000"
+       ],
+       "X-Forwarded-Proto": [
+         "http"
+       ],
+       "X-Request-Id": [
+         "d645dc37-5326-4b69-8c2c-4060e12ca4ff"
+       ]
+     },
+     "origin": "10.0.64.27:53026",
+     "url": "http://delegation.example:8080/anything/team2/bar",
+     "data": "",
+     "files": null,
+     "form": null,
+     "json": null
+   }
+   ```
+   
+## Use wildcard namespaces {#wildcard}
+
+Instead of routing to an HTTPRoute with a specific label in a specific namespace, you can use a wildcard for the namespace. This configuration can streamline your route delegation setup as it allows you to easily add new child HTTPRoutes to the delegation chain. 
```

Comment:
```suggestion
Instead of routing to an HTTPRoute with a specific label in a specific namespace, you can use a wildcard for the namespace. This configuration can streamline your route delegation setup, as it allows you to easily add new child HTTPRoutes to the delegation chain. 
```